### PR TITLE
[Release] decK version 1.17

### PR DESCRIPTION
**Pull request by Guaris**
This PR bumps the decK version to from 1.16 to 1.17. 

Autodocs do not need to be run for decK 1.17

### CODE REVIEW

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Not sure if we're waiting to merge this PR until all the decK 1.17 stuff is also merged in, so I'll hold off on merging this one

**Comment by acgoldsmith** [comment]
Tied to DOCU-2810

File: app/_data/kong_versions.yml
Comment (mheap) [inline app/_data/kong_versions.yml:252]:
```suggestion
  version: "1.17.0"
```

**Review by GGabriele — APPROVED** [review]

**Pull request merged**
